GoogleDataTransport/GDTCORLibrary/GDTCORTransport.m

```diff
@@ -93,4 +93,16 @@ - (void)sendEvent:(GDTCOREvent *)event
                                 onComplete:completion];
 }
 
+#pragma mark - Force Category Linking
+
+extern void GDTCORInclude_GDTCORLogSourceMetrics_Internal_Category(void);
+
+/// Does nothing when called, and not meant to be called.
+///
+/// This method forces the linker to include categories even if
+/// users do not include the '-ObjC' linker flag in their project.
++ (void)noop {
```

Comment:
Why a 2nd noop function?

Reply:
Some of the categories are in 
- GoogleDataTransport/GDTCORLibrary
and the other are in
- GoogleDataTransport/GDTCCTLibrary

GDTCCTLibrary is supposed to sit on top of GDTCORLibrary in case it was ever refactored into a separate module, so I added a noop for the categories under each directory.

Reply:
So I was trying to make this future-proof. But, that being said, a refactor is probably unlikely so I could consolidate if you'd prefer.

Reply:
I'm ok like this and we can address it if and when we ever refactor.

Reply:
SG, thanks for the review!